src/status_im/contexts/wallet/send/routes/style.cljs

```diff
@@ -12,30 +12,32 @@
   {:flex-direction  :row
    :justify-content :space-between})
 
-(def routes-inner-container
-  {:margin-top      8
+(defn routes-inner-container
+  [first-item?]
+  {:margin-top      (if first-item? 7.5 11)
```

Comment:
do we use .5 margins?  🤔

Reply:
Yeah there are many places, it can be sometimes noticeable depending on the device dpi. In this case is because of this:
![](https://github.com/status-im/status-mobile/assets/18485527/da907fea-05b3-4e96-8dd4-30e14bb6ec29)